### ISSUE TITLE
504 error handling - black on blac - a11y - color contrast 

### ISSUE THREAD

**Issue by ioana-chiorean** [codeeu/codeweek#1977]
Got 504 and i realized a barely see anything. Funny enough my name in informal Romanian means the blind one.
Can we improve the a11y on this? 
<img width="1699" alt="Screenshot 2022-10-27 at 23 22 55" src="https://user-images.githubusercontent.com/1391451/198402288-72d3b9d0-eb36-47e3-90ca-18416092ac57.png">

**Comment by alainvd** [comment]
The message is displayed by the browser and is not styled by codeweek website. I'm afraid I can't do much.
The 504 were caused by abnormal heavy load on the website. This should be fixed.
